CHANGELOG.md

```diff
@@ -119,5 +119,110 @@ This option is new for VectorCAST 23 sp2
 - Added support for relative paths to the coded test file in test script #106
 - Fixed issue with unrecoverable link error after a coded test compile error #128
 
+## [1.0.11] - 2024-06-24
 
+### Code Maintenance
+- Increased timeouts for E2E tests.
+
+## [1.0.11] - 2024-06-27
+
+### Code Maintenance
+- Split E2E tests and isolated deletion tests.
```

Comment:
We can get rid of 130/131

Reply:
Deleted in 6d05d2f02c29badcfa0e210b76118c22d8908731.